assessment.md

```diff
@@ -143,39 +143,96 @@ TODO: add description for each assessment level for this skill.
 
 * Software documentation
 
-TODO: add description for each assessment level for this skill.
+Software Documentation will be evaluated on this scale
+
+* None = No explanation of what the software does, no solution to the problem presented, and no example of where the code be used is included
+* Inadequate = A short explanation, no solution to the problem, and no example is included
+* Adequate = A good explanation, a short solution to the problem, and a short example is included
+* Good = A detailed explanation, a solution to the problem, a short example is included
+* Excellent = A detailed explanation, a detailed solution to the problem, a interactive example is included
 
 * Programming styles
 
-TODO: add description for each assessment level for this skill.
+Programming styles will be evaluated on the following levels:
+
+* None = Does not have the ability to program at all in any style.
+* Inadequate = Programming style is completely akin to "spaghetti code."
+* Adequate = Programming style is slightly akin to "spaghetti code." with some correct naming conventions, indentations, and little or non-descriptive commenting.
+* Good = Programming style is fairly thorough and mostly uses proper naming conventions, indentations, and a fair amount of commenting. Styling is mostly consistent at this level.
+* Excellent = Programming style is very thorough and uses proper naming conventions, indentations, and is descriptively commented throughout. Styling is consistent at this level.
 
 * Managing software complexity
 
-TODO: add description for each assessment level for this skill.
+Managing software complexity will be evaluated on the following scale:
+
+* None = Written software is at a state where once the author has left the code, it is quite possible they would not be able to recognize what they left behind. Complexities reach a level so high that code becomes illegible to any reader.
+* Inadequate = Written software is not efficient in nature and often strays into unnecessary complexities; conditional statements are frequently overly convoluted and hard-coding is consistently present throughout.
+* Adequate = Written software is hardly efficient in nature and occasionally strays into unnecessary complexities; conditional statements can occasionally be overly convoluted and hard-coding is frequently present.
+* Good = Written software is decently efficient in nature and barely strays into unnecessary complexities; conditional statements are often not overly convoluted and hard-coding is only semi-present at best.
+* Excellent = Written software is incredibly efficient in nature and does not stray into unnecessary complexities or perhaps enter complexity without reason; conditional statements are not overly convoluted and hard-coding is hardly, if at all present.
 
 * Mitigating software risk
 
-TODO: add description for each assessment level for this skill.
+Ability to plan for and mitigate risks.
+
```

Comment:
These two lines can be removed for consistency with other sections of the document